### PR TITLE
Hyperledger Indy Project Report 2023Q4

### PR DESCRIPTION
**Pull request by swcurran**
@TelegramSam @cjhowland @WadeBarnes @lynnbendixsen @Toktar

### CODE REVIEW

File: project-reports/2023/2023-Q4-Hyperledger-Indy.md
Comment (tkuhrt) [inline project-reports/2023/2023-Q4-Hyperledger-Indy.md:153]: Is there a Indy-specific maintainer policy, or could we use the [Maintainer Inactivity Policy](https://toc.hyperledger.org/governing-documents/inactivity.html#maintainer-inactivity) that the TOC previously approved for determining when the last action was taken and removing anyone that has not had activity in the past 6 months?

**Review by tkuhrt — APPROVED** [review]

File: project-reports/2023/2023-Q4-Hyperledger-Indy.md
Comment (swcurran) [reply project-reports/2023/2023-Q4-Hyperledger-Indy.md:153]: Yes.  It’s just been time and priorities that has prevented this from being done...

**Review by petermetz — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

File: project-reports/2023/2023-Q4-Hyperledger-Indy.md
Comment (lehors) [inline project-reports/2023/2023-Q4-Hyperledger-Indy.md]:
```suggestion
then use those funds to enable a code contribution?
```

**Review by lehors — APPROVED** [review]

**Review by bobbi28 — APPROVED** [review]

File: project-reports/2023/2023-Q4-Hyperledger-Indy.md
Comment (VRamakrishna) [inline project-reports/2023/2023-Q4-Hyperledger-Indy.md:89]: "and the use" --> "and then use"

File: project-reports/2023/2023-Q4-Hyperledger-Indy.md
Comment (VRamakrishna) [inline project-reports/2023/2023-Q4-Hyperledger-Indy.md:125]: Why the "should"? Are there drawbacks in the Indy permission model?

If you can port Indy objects to Besu, I imagine it should not be too hard to port it to any other kind of ledger, right?

**Review by VRamakrishna — APPROVED** [review]

File: project-reports/2023/2023-Q4-Hyperledger-Indy.md
Comment (swcurran) [reply project-reports/2023/2023-Q4-Hyperledger-Indy.md:125]: The “should” is just there in case it is difficult to achieve and there are not sufficient benefits. The community has adapted to the Indy roles/permission model, and definitely works. However, I didn’t want to be so rigid as to say “It must be exactly the same or we walk away”.  I wanted to leave room for other ideas, and potential learnings from Besu and its deployments.  That said — in subsequent discussions with some Besu operators, I see that the permissions/roles are quite similar.

The objects on Indy are the objects of Hyperledger AnonCreds, and that project has already demonstrated that those objects can be stored on other types of ledgers. The point of moving AnonCreds out of Indy was to promote that the technology is ledger-agnostic, and to make it easy to root AnonCreds credentials on any ledger (and even non-ledger) storage.

The goal of an “Indy on Besu” approach is to provide a transition from current Indy ledgers to future Indy ledgers — providing comparable tooling and capabilities that exist for Indy to others wanting to run a ledger that has first class support for privacy-preserving verifiable credentials. The benefit of that goal is that ANY Besu network would gain that “first class support”.

**Review by jimthematrix — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Pull request merged**
